### ISSUE TITLE
loose popup track when clicking "voir l'itinéraire"

### ISSUE THREAD

**Issue by drzraf** [gdesmott/manger-veggie#17]
When clicking the link to "see the route", the current window is used and the popup, zoom level and location are lost when we get back.
Either:
- a new window/new popup/new xxx can be opened so ones stays free to browse the map at the meantime (I'd prefer this solution, especially since you can't easily "open in a new tab" using a smartphone)
- a the state of the map browsing could be tracked and kept Keeping a track means a different output from scripts according to HTTP parameters (URL parameters, cookies, ...) which may make browser caching more difficult.
The data-API may be the good tool for the task. Maybe it'd be worth the case if other features would benefit from this state storage *and* leaflet provide helpers for the task.

**Comment by Psycojoker** [comment]
I think that both ideas make sens actually, for different use case:

* the popup solve the problem you've mentionned and it's easy to implement (add target="_blank" to the a tag)
* the other one will be good for problems like accidently closing the tag and so one

**Comment by gdesmott** [comment]
I think I'd prefer the new popup/window approach for this specific case. Be carreful to have something working with the Android app as well.

The satefull approach can still be useful, when viewing the details of a restaurant and the going back for example. But maybe it's not worth our time and we should focus implementing issue #10 instead.

**Comment by Psycojoker** [comment]
I agree that #10 would be better, we "just" need someone to make it :/
